### ISSUE TITLE
Rename rest of package to LuckyInflector

### ISSUE THREAD

**Issue by paulcsmith** [luckyframework/inflector#5]
Need to figure out how to handle this namespacing though because it looks weird: `LuckyInflector::Inflector`...but maybe it's fine

**Comment by actsasflinn** [comment]
I'm using it right now and it feels a little weird.

**Comment by paulcsmith** [comment]
Yeah it does look a bit weird. Any ideas for a better module name?

**Comment by paulcsmith** [comment]
Maybe move everything could move from LuckyInflector::Inflector to just LuckyInflector. Example: LuckyInflector.pluralize(...)

**Comment by actsasflinn** [comment]
Are you trying not to have a LuckySupport? I think there could be some other useful utility type libs that would be useful through the Record, Web, etc.

I get the sense you don’t want to extend Crystal in the way Rails extended Ruby with ActiveSupport. Am I reading that right?

**Comment by paulcsmith** [comment]
I actually like ActiveSupport, but I don't like that it forces you to extend core classes. I know a lot of people don't like that and I'd love it if people could not do that so they can use all the goodies :)

I agree that there are other useful util libs, but someone pointed out that inflector is a good thing to have on its own w/o needing to include everything. I think instead we could have a `LuckyUtils` lib that has number helpers, time helpers, etc. That way people can use just the inflector if that's what they want. Does that make sense?

**Comment by actsasflinn** [comment]
Yeah, I dig it. There’s been times I’ve wanted just one thing from ActiveSupport for a Sinatra app or a command line map/reduce script but I hate bringing in a truckload of lib for a method or two.

Re #5, yeah that’s a bit better. I’d also like to be able to call it stanatalone or include it.

    class Foo
      include LuckyInflector

      def blah
        pluarlize(“blah”)
      end
    end
